packages/wallet-backend-update/src/rapyd/rapyd-client.ts

```diff
@@ -159,6 +174,29 @@ export class RapydClient implements IRapydClient {
     return this.get<RapydResponse<RapydCountry[]>>('data/countries')
   }
 
+  public getPayoutMethodTypes(
+    assetCode: string
+  ): Promise<RapydResponse<PayoutMethodResponse[]>> {
+    return this.get(
+      `payouts/supported_types?payout_currency=${assetCode}&limit=1`
+    )
+  }
+
+  public withdrawFundsFromAccount(
+    req: WithdrawFundsFromAccountRequest
+  ): Promise<RapydResponse<WithdrawFundsFromAccountResponse>> {
+    return this.post('payouts', JSON.stringify(req))
+  }
+
+  public completePayout(
```

Comment:
so in here, I think completePayout should be private, and used inside of withdrawFunds. If the first withdraw funds call is sucessfull, then completePayout will be called further. and if that is sucessfull as well, then return.